binding-library/java/src/main/java/com/microsoft/azure/functions/kafka/annotation/KafkaOutput.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.kafka.annotation;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+import com.microsoft.azure.functions.kafka.BrokerAuthenticationMode;
+import com.microsoft.azure.functions.kafka.BrokerProtocol;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.PARAMETER)
+@Retention(RetentionPolicy.RUNTIME)
+@CustomBinding(direction = "in", name = "", type = "KafkaOutput")
```

Comment:
direction needs to be `out`

Reply:
Sorry, I forget to push the new version. My current is added.